### ISSUE TITLE
The learning rate has never been updated by lr_scheduler

### ISSUE THREAD

**Issue by HcPlu** [aai-institute/tianshou#774]
- [x] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [x] RL algorithm bug
    + [ ] documentation request (i.e. "X is missing from the documentation.")
    + [ ] new feature request
- [x] I have visited the [source website](https://github.com/thu-ml/tianshou/)
- [x] I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
- [x] I have mentioned version numbers, operating system and environment, where applicable:

I have logged the learning rate during training.  The invariant learning rate is observed even thought equipped with a lr scheduler following the given example.

This bug happens at the `learn()` function of `PPOPolicy`.
The following sentences are missing:
  ```python
        if self.lr_scheduler is not None:
            self.lr_scheduler.step()
  ```

**Comment by Trinkle23897** [comment]
It's in BasePolicy.
